src/pages/graphql/schema/orders/mutations/confirm-return.md

```diff
@@ -0,0 +1,135 @@
+---
+title: confirmReturn mutation
+edition: ee
+---
+
+# confirmReturn mutation
+
```

Comment:
Add the following block below this blank line. (Code gates added to prevent GH from trying render it.)

```
<InlineAlert variant="info" slots="text1" />

This mutation is part of the [Storefront Compatibility Package](https://experienceleague.adobe.com/developer/commerce/storefront/setup/storefront-compatibility/). It will be added to Adobe Commerce 2.4.8-beta2.
```